vault/src/issue.rs

```diff
@@ -92,23 +92,54 @@ pub async fn add_keys_from_past_issue_request<B: BitcoinCoreApi + Clone + Send +
         None => return Ok(()), // the iterator is empty so we have nothing to do
     };
 
-    for (issue_id, request) in issue_requests.into_iter() {
+    for (issue_id, request) in issue_requests.clone().into_iter() {
         if let Err(e) = add_new_deposit_key(bitcoin_core, issue_id, request.btc_public_key).await {
             tracing::error!("Failed to add deposit key #{}: {}", issue_id, e.to_string());
         }
+        // TODO: get TXs from elects
+        // then importprunedfunds
```

Comment:
This seems like something we want in this PR right?

Reply:
Whoops, old dev comment. (We do want that - and it's already implemented of course, haha.)